serverless.yml

```diff
@@ -7,22 +7,95 @@ plugins:
   - serverless-step-functions
   - serverless-pseudo-parameters
 
+custom:
+  DYNAMO_DB_TABLE: stackJanitorTable
+  DEFAULT_EXPIRATION_PERIOD: 604800 #seconds
+
 provider:
   name: aws
   runtime: nodejs10.x
+  memorySize: 128
+  stage: ${opt:stage, 'dev'}
   region: ap-southeast-2
   iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "dynamodb:DescribeTable"
+        - "dynamodb:Query"
+        - "dynamodb:Scan"
+        - "dynamodb:GetItem"
+        - "dynamodb:PutItem"
+        - "dynamodb:UpdateItem"
+        - "dynamodb:DeleteItem"
+      Resource: "arn:aws:dynamodb:*:*:table/${self:custom.DYNAMO_DB_TABLE}"
     - Effect: "Allow"
       Action:
         - "cloudformation:DescribeStack*"
+        - "cloudformation:DeleteStack*"
+      Resource: "*"
+    - Effect: "Allow"
+      Action:
+        - "ec2:*"
+        - "s3:*"
+        - "ecs:*"
+        - "elasticloadbalancing:*"
+        - "elasticloadbalancing:*"
+        - "elasticloadbalancingv2:*"
+        - "logs:*"
```

Comment:
I think it would be worth investing some time ensuring that this only has access to delete permissions. Could you raise a ticket for this?

Reply:
Ok. I will